### PR TITLE
Add Radio Input

### PR DESCRIPTION
**Pull request by Johannafendrich**
Storybook preview:
https://my-homie-radio-zieqzl7xfglqqh4.herokuapp.com/storybook/index.html?path=/story/radio-input--radio

### CODE REVIEW

File: client/src/components/RadioInput.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import PropTypes from 'prop-types';
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
```
Comment (hauptdigital) [inline client/src/components/RadioInput.js:8]:
```suggestion
  justify-content: left;
```
Overall good development 👍 

The only thing is that in your preview in storybook, the radiobuttons and labels do not seem to be aligned. I don't know the overall context but I added a suggestion to align your labels to the left.

File: client/src/components/RadioInput.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import PropTypes from 'prop-types';
+
+const Container = styled.div`
```
Comment (hauptdigital) [inline client/src/components/RadioInput.js:5]: This looks like a generic component that might be useful in other places of the app in the future. Maybe it makes sense to move this into a new file.

File: client/src/components/RadioInput.js (hunk 1 above)
Comment (Johannafendrich) [reply client/src/components/RadioInput.js]: Thanks for your tip. It worked out great with `justify-content: left` 👌🏼

File: client/src/components/RadioInput.js (hunk 2 above)
Comment (Johannafendrich) [reply client/src/components/RadioInput.js]: This is true and I will use it as a separate component in another place 🤗

**Pull request merged**